Exception on startup

```
java.lang.RuntimeException: Exception in Application start method
    at javafx.graphics/com.sun.javafx.application.LauncherImpl.launchApplication1(Unknown Source)
    at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(Unknown Source)
    at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NullPointerException
    at com.faforever.client.FafClientApplication.showMainWindow(FafClientApplication.java:92)
    at com.faforever.client.FafClientApplication.start(FafClientApplication.java:77)
    at javafx.graphics/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(Unknown Source)
    at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$11(Unknown Source)
    at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$9(Unknown Source)
    at java.base/java.security.AccessController.doPrivileged(Native Method)
    at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(Unknown Source)
    at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(Unknown Source)
    at javafx.graphics/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
    at javafx.graphics/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(Unknown Source)
    ... 1 more
```

--- a/src/main/java/com/faforever/client/FafClientApplication.java
+++ b/src/main/java/com/faforever/client/FafClientApplication.java
@@ -8,6 +8,7 @@ import com.faforever.client.preferences.PreferencesService;
 import com.faforever.client.theme.UiService;
 import com.faforever.client.ui.StageHolder;
 import com.faforever.client.ui.taskbar.WindowsTaskbarProgressUpdater;
+import com.github.nocatch.NoCatch.NoCatchRunnable;
 import javafx.application.Application;
 import javafx.application.Platform;
 import javafx.scene.text.Font;
@@ -24,6 +25,9 @@ import org.springframework.context.annotation.Bean;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.CountDownLatch;
+
+import static com.github.nocatch.NoCatch.noCatch;
 
 @SpringBootApplication(exclude = {
     JmxAutoConfiguration.class,
@@ -64,10 +68,17 @@ public class FafClientApplication extends Application {
     Font.loadFont(FafClientApplication.class.getResourceAsStream("/font/dfc-icons.ttf"), 10);
     JavaFxUtil.fixTooltipDuration();
 
-    Platform.runLater(() -> applicationContext = new SpringApplicationBuilder(FafClientApplication.class)
-        .profiles(getAdditionalProfiles())
-        .bannerMode(Mode.OFF)
-        .run(getParameters().getRaw().toArray(new String[0])));
+    CountDownLatch runAndWaitLatch = new CountDownLatch(1);
+
+    Platform.runLater(() -> {
+      applicationContext = new SpringApplicationBuilder(FafClientApplication.class)
+          .profiles(getAdditionalProfiles())
+          .bannerMode(Mode.OFF)
+          .run(getParameters().getRaw().toArray(new String[0]));
+      runAndWaitLatch.countDown();
+    });
+
+    noCatch((NoCatchRunnable) runAndWaitLatch::await);
   }
 
   @Override

